src/extension/wps-download/src/main/java/org/geoserver/wps/gs/download/DownloadAnimationProcess.java

```diff
@@ -175,8 +193,19 @@ public DownloadAnimationProcess(
             AWTSequenceEncoder enc = new AWTSequenceEncoder(out, frameRate);
 
             DownloadServiceConfiguration configuration = confiGenerator.getConfiguration();
-            TimeParser timeParser = new TimeParser(configuration.getMaxAnimationFrames());
-            Collection parsedTimes = timeParser.parse(time);
+            int totalFrames;
+            Collection parsedTimes = null;
+            List<String> animateValuesList = new ArrayList<>();
+            if (!StringUtils.isEmpty(time)) {
```

Comment:
The documentation states time and animateParam are incompatible, better to check if they are both specified and throw an exception.